sbin/root-mount

```diff
@@ -161,6 +168,11 @@ check_firstboot_resize
 # TODO: revisit for encryption, where home would not be mounted here.
 if (mount -t ext4 $ROOTDEV $1 && mount -t ext4 $HOMEDEV $1/home); then
 	echo "root_mount: Root and home mounted" > /dev/kmsg
+	# If we are here then filesystem is already resized.
+	if ! test -f "$1/$FS_RESIZED"; then
+		> "$1/$FS_RESIZED"
```

Comment:
We end up here only once after flashing or factory reset so I would keep it simple and use touch if available in initrd.